### PR TITLE
[CI] test hierarchy builder as part of elpi

### PR DESCRIPTION
**Pull request by gares**

### CODE REVIEW

File: dev/ci/ci-basic-overlay.sh [hunk 1]
```diff
@@ -214,11 +214,14 @@
 : "${equations_CI_ARCHIVEURL:=${equations_CI_GITURL}/archive}"
 
 ########################################################################
-# Elpi
+# Elpi + Hierarchy Builder
 ########################################################################
 : "${elpi_CI_REF:=coq-master}"
 : "${elpi_CI_GITURL:=https://github.com/LPCIC/coq-elpi}"
 : "${elpi_CI_ARCHIVEURL:=${elpi_CI_GITURL}/archive}"
+: "${elpi_hb_CI_REF:=coq-master}"
```
Comment (SkySkimmer) [inline dev/ci/ci-basic-overlay.sh]: Please leave a space
```suggestion

: "${elpi_hb_CI_REF:=coq-master}"
```

File: dev/ci/ci-elpi.sh [hunk 2]
```diff
@@ -6,3 +6,7 @@ ci_dir="$(dirname "$0")"
 git_download elpi
 
 ( cd "${CI_BUILD_DIR}/elpi" && make && make install )
+
+git_download elpi_hb
+
+( cd "${CI_BUILD_DIR}/elpi_hb" && make && make install )
```
Comment (SkySkimmer) [inline dev/ci/ci-elpi.sh]: One the one hand I would prefer if it was a separate script
On the other hand that would require changing the job name on the gitlab side so maybe not worth it

**Comment by gares** [comment]
done thanks

**Comment by gares** [comment]
CI passed, can someone assign?

**Review by SkySkimmer — APPROVED** [review]

**Comment by gares** [comment]
Thanks

**Pull request merged**